atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -107,6 +116,16 @@ public void conservativeSweepAddsSentinelAndLeavesSingleValue() {
         assertTestValueEnqueuedAtGivenTimestampStillPresent(TABLE_CONS, LOW_TS);
     }
 
+    @Test
+    public void sweepWithSingleEntryUpdatesMetrics() {
+        enqueueWrite(TABLE_CONS, LOW_TS);
+        sweepQueue.sweepNextBatch(ShardAndStrategy.conservative(CONS_SHARD));
+
+        TargetedSweepMetricsTest.assertTombstonesPutConservativeEquals(1);
```

Comment:
No, you put a tombstone even if you only had one write since targeted sweep does not maintain state on previous writes that were processed. In the case there is only a single write in the cell, the tombstone will not actually delete anything, but we still put it.